tests/test_image_transforms.py

```diff
@@ -313,6 +322,46 @@ def test_normalize(self):
             np.allclose(normalize(image, mean=mean, std=std, input_data_format="channels_last"), expected_image)
         )
 
+    @parameterized.expand(
+        [
+            ((3, 224, 224), (120, 60)),
+            ((3, 223, 223), (120, 60)),
+            ((3, 224, 224), (121, 61)),
+            ((3, 223, 223), (121, 61)),
+        ]
+    )
+    @require_torchvision
+    def test_center_crop_with_torchvision(self, image_size, expected_size):
```

Comment:
This name of this test is a bit misleading - torchvision is used as a reference but we're testing center crop with our own functionality. I'd just merge this logic with `test_center_crop` and have a single method.

Reply:
As the original function `test_center_crop()` performs other types of tests (like crop an area larger than the original image, 4-channel image), I left this extra test in a separate function named `test_center_crop_for_various_sizes`, and removed the torchvision dependency.
Please, let me know if it's ok.